src/components/user/user.service.ts

```diff
@@ -200,6 +200,16 @@ export class UserService {
     return true;
   }
 
+  async isEmailUnique(id: string, session: ISession): Promise<boolean> {
+    const isUnique = await this.db.isPropertyUnique({
+      id,
+      session,
+      baseNodeLabel: 'User',
+      propName: 'email',
+    });
+    return isUnique;
+  }
```

Comment:
We need to enforce the user's email is unique before the user is created.

The only other time this info would be helpful is for the frontend to check for the email before submitting. And in that case they don't have a user ID, only an email address.

Reply:
this isn't a ticket for checking Uniqueness or email.  This was for the uniqueness function in db-service.  The email uniqueness was an example michael wrote in the ticket, so it was the tester function that allowed Koijam to run an e2e test through the db-service.  
the utility of checking email uniqueness is probably low.  Email uniqueness is already a DB constraint.

Reply:
Ah gotcha. So we don't need the resolver then. Just call the service method directly in test.